clinica/utils/io.py

```diff
@@ -145,12 +157,35 @@ def check_bids_folder(bids_directory):
              Fore.BLUE, bids_directory, Fore.RESET)
         )
 
+    if isdir(join(bids_directory, 'subjects')):
+        raise ClinicaBIDSError(Fore.RED + '\n[Error] The BIDS directory (' + bids_directory + ') you provided seems to '
+                               + 'be a CAPS directory due to the presence of a \'subjects\' folder.' + Fore.RESET)
+
+    if len(listdir(bids_directory)) == 0:
+        raise ClinicaBIDSError(Fore.RED + '\n[Error] The BIDS directory you provided  is empty. (' + bids_directory
+                               + ').' + Fore.RESET)
+
+    if len([item for item in listdir(bids_directory) if item.startswith('sub-')]) == 0:
+        raise ClinicaBIDSError(Fore.RED + '\n[Error] Your BIDS directory does not contains a single folder whose name '
+                               + 'starts with \'sub-\'. Check that your folder follow BIDS standard' + Fore.RESET)
+
 
 def check_caps_folder(caps_directory):
+    """
+    check_caps_folder function checks the following items :
```

Comment:
`the following items :` -> `the following items:` (English formatting)

Reply:
Corrected